webapp/src/components/App/Singlestudy/explore/Modelization/Areas/Storages/StorageConfig/index.tsx

```diff
@@ -38,16 +40,27 @@ function StorageConfig() {
   return (
     <>
       <BackButton onClick={() => navigate("../storages")} />
-      <StorageForm study={study} areaId={areaId} storageId={storageId} />
-      <Divider sx={{ my: 2 }} variant="middle">
-        <Chip label={t("global.matrices")} size="small" />
-      </Divider>
-      <StorageMatrices
-        study={study}
-        areaId={areaId}
-        storageId={nameToId(storageId)}
-        studyVersion={studyVersion}
-      />
```

Comment:
@skamril sorry but I didn't notice this removes the Matrices from the config form, this has to be put back please

Reply:
don't know if a tab for the matrices is acceptable for users, if not we can create a component that combines config form + matrices